Source/Plugins/Statistics/StatisticsFilters/FindShapes.cpp

```diff
@@ -36,10 +36,12 @@
 #include "FindShapes.h"
 
 #include <array>
+#include <cmath>
```

Comment:
```suggestion
#include <cmath>
#include <utility>
```
For std::swap below.